ozaria/site/styles/play/level/tome/problem_alert.sass

```diff
@@ -8,16 +8,26 @@
   // Position these at the end of the spell editor, right above the spell toolbar.
   top: 45px
   right: 500px
-  background: transparent
-  border: 1px solid transparent
-  padding: 0
-  font-size: 18px
+  background-color: #ffffff
+  padding: 18.5px 13px
+  font-size: 16px
+  font-family: "Open Sans"
+  letter-spacing: 0.63px
+  line-height: 24px
   text-shadow: none
-  color: white
+  color: #232323
```

Comment:
Discussion point and not blocking.

Is this a color we reuse within Ozaria? If so should we store it as a variable in an ozaria sass variables file?

Reply:
Yep I think its re-usable, will check and update in the next PR